src/System.ComponentModel.Primitives/src/project.json

```diff
@@ -1,11 +1,24 @@
 {
   "frameworks": {
+    "netstandard1.7": {
+      "dependencies": {
+        "System.Collections.NonGeneric": "4.3.0-beta-24522-03",
+        "System.ComponentModel": "4.3.0-beta-24522-03",
+        "System.Diagnostics.Tools": "4.3.0-beta-24522-03",
+        "System.Resources.ResourceManager": "4.3.0-beta-24522-03",
+        "System.Runtime": "4.3.0-beta-24522-03",
+        "System.Threading": "4.0.0"
```

Comment:
Bump System.Threading package version up to the latest 4.3.0-beta-* version as well, otherwise you will be seeing downgrade warnings.